server/vulnerabilities/nvd/sync/cve_syncer.go

```diff
@@ -4,21 +4,27 @@
 package nvdsync
```

Comment:
Some small nits but to avoid confusion in the future:
- Update `nvdsync` package documentation.
- Update the `CVE` type documentation in this package.